### PR TITLE
Add details option for GET /containers/{id}/logs

### PR DESCRIPTION
**Pull request by yuting-liu**
In page `Engine API version history` (https://docs.docker.com/engine/api/version-history/), there is a new `v1.23 API change` for `GET /containers/(id or name)/logs`, which now accepts a `details` query parameter to stream the extra attributes that were provided to the containers `LogOpts`, such as environment variables and labels, with the logs.
This `details` is a boolean option. If `true`, extra details provided to logs will be shown. The default value is `false`.

---
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/965)

### CODE REVIEW

File: src/main/java/com/github/dockerjava/api/command/LogContainerCmd.java [hunk 1]
```diff
@@ -69,6 +74,8 @@
 
     LogContainerCmd withSince(Integer since);
 
+    LogContainerCmd withDetails(Boolean details);
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/LogContainerCmd.java]: Please add javadoc `@since 1.23`

**Comment by KostyaSha** [comment]
Could you add this verification into integration test?

**Comment by yuting-liu** [comment]
Hi @KostyaSha,
Do I need to add a separate method `asyncLogContainerWithSince` into integration test? or just used directly in `asyncLogContainerWithTtyDisabled` and `asyncLogContainerWithTtyEnabled`?

**Comment by KostyaSha** [comment]
better separate

**Comment by yuting-liu** [comment]
@KostyaSha I'm sorry that the branch of this PR is accidentally deleted after my forked repo is deleted. So I created a new PR (https://github.com/docker-java/docker-java/pull/967) with integration tests added.

**Pull request closed without merging**